subsys/net/lib/lwm2m/Kconfig

```diff
@@ -186,6 +186,11 @@ config LWM2M_QUEUE_MODE_UPTIME
 	  defaults to 93 seconds, see RFC 7252), it does not forbid other
 	  values though.
 
+config LWM2M_TLS_SESSION_CACHING
+	bool "Enable TLS session caching"
+	help
+	  Enable this only when feature is supported.
```

Comment:
```suggestion
	  Enable this only when feature is supported in Mbed TLS.
```

Reply:
I think we should not mention mbed TLS originally I have it "supported in modem"

Reply:
It could be also just "in TLS library", currently it is a bit vague in my opinion.